raml/APIs/SecuritySchemes.raml

```diff
@@ -0,0 +1,36 @@
+#%RAML 1.0 securitySchemes
+basic:
+  type: Basic Authentication
+  description: |
+    This API supports Basic Authentication.
+
+oauth_2_0:
```

Comment:
Wondering why this is here? Surely this only would be needed on resource servers?